sentry-core/src/main/java/io/sentry/core/SentryEvent.java

```diff
@@ -304,4 +304,20 @@ public DebugMeta getDebugMeta() {
   public void setDebugMeta(DebugMeta debugMeta) {
     this.debugMeta = debugMeta;
   }
+
+  // Level is Fatal or any exception was unhandled by the user.
+  public boolean isCrashed() {
+    if (level == SentryLevel.FATAL) {
+      return true;
+    }
+    if (exception != null) {
+      for (SentryException e: exception.getValues()) {
+        if (e.getMechanism() != null && !e.getMechanism().isHandled()) {
+          return true;
+        }
+      }
```

Comment:
does it make sense to add a `transient` field to` SentryEvent` if the event is handled or not? so when adding exceptions to the event, as we know when it's not handled, we could read directly from there, we would spare this `for`, but with the overhead of having in both places.

Reply:
Maybe something to discuss in the TSC. So far it's an agreement based on either case